content/es/what-is-opentelemetry.md

```diff
@@ -0,0 +1,132 @@
+  Uno de los objetivos principales de OpenTelemetry es que puedas instrumentar
+  fácilmente tus aplicaciones o sistemas, sin importar el lenguaje de
+  programación, infraestructura o entorno de ejecución. El almacenamiento y
+  visualización de la telemetría se deja intencionalmente a otras herramientas.
+
+## ¿Qué es la observabilidad?
+
+[Observabilidad](/docs/concepts/observability-primer/#what-is-observability) es
+la capacidad para comprender el estado interno de un sistema a través del
+análisis de sus resultados. En el contexto del software, esto significa ser
+capaces de comprender el estado interno de un sistema a través del análisis de
+sus datos de telemetría, que incluyen trazas, métricas y logs.
+
+Para que un sistema sea observable, debe ser
+[instrumentado](/docs/concepts/instrumentation), esto es, su código debe emitir
+[trazas](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) o
+[logs](/docs/concepts/signals/logs/). Los datos instrumentados tienen que ser
+enviados luego a un backend de observabilidad.
+
+## ¿Por qué OpenTelemetry?
+
+Con el auge de la computación en la nube, las arquitecturas de microservicios y
+requisitos comerciales cada vez más complejos, la necesidad de
+[observabilidad](/docs/concepts/observability-primer/#what-is-observability) del
+software y la infraestructura es mayor que nunca.
+
+OpenTelemetry satisface la necesidad de observabilidad siguiendo dos principios
+fundamentales:
+
+1. Eres dueño de los datos que generas. No hay ningún tipo de dependencia del
+   proveedor.
+2. Solo tienes que aprender un único conjunto de API y convenciones.
+
+Ambos principios combinados otorgan a los equipos y organizaciones la
+flexibilidad que necesitan en el mundo informático de hoy en día.
+
+Si quieres aprender más, echa un vistazo a
+[la misión, la visión y los valores](/community/mission/) de OpenTelemetry.
+
+## Main OpenTelemetry components
+
+OpenTelemetry consta de los siguientes componentes principales:
+
+- Una [especificación](/docs/specs/otel) para todos los componentes.
+- Un [protocolo](/docs/specs/otlp/) estándar que define el formato de los datos
+  de telemetría.
+- Una [convención semántica](/docs/specs/semconv/) que define un sistema de
+  nomenclatura estándar para tipos de datos de telemetría comunes.
+- APIs que describen cómo se generan los datos de telemetría.
+- [SDKs para cada lenguaje](/docs/languages) que implementan las
+  especificaciones, las APIs, y que exportan datos de telemetría.
+- Un [ecosistema de librerías](/ecosystem/registry) que implementan
+  instrumentación para librerías y frameworks populares.
+- Componentes de instrumentación automática que generan datos de telemetría sin
+  requerir cambios en el código.
+- El [OpenTelemetry Collector](/docs/collector), un proxy que recibe, procesa y
+  exporta datos de telemetría.
+- Numerosas herramientas, como el
+  [OpenTelemetry Operator for Kubernetes](/docs/kubernetes/operator/),
```

Comment:
Not sure if we should translate that or not?
```suggestion
  [OpenTelemetry Operator para Kubernetes](/docs/kubernetes/operator/),
```

Reply:
We should! It's a title, not a product or feature name